apps/passport-client/src/dispatch.ts

```diff
@@ -598,42 +590,34 @@ async function sync(state: AppState, update: ZuUpdate) {
       downloadingPCDs: true
     });
 
-    /**
-     * Get both PCDs and subscriptions
-     * Subscriptions might be null even if we got PCDs, if we were
-     * downloading from a pre-v3 version of encrypted storage
-     * {@link SyncedEncryptedStorageV3}
-     * */
-    const downloaded = await downloadStorage();
-
-    if (downloaded != null && downloaded.pcds != null) {
-      const { pcds, subscriptions } = downloaded;
-
-      if (subscriptions) {
-        addDefaultSubscriptions(subscriptions);
-      }
+    // Download user's E2EE storage, which includes both PCDs and subscriptions.
```

Comment:
@robknight Here's the top-level of the code change based on our discussion of subscription syncing.  Please take a look and let me know if I missed anything.